docs/reference/commandline/dockerd.md

```diff
@@ -1147,6 +1147,7 @@ This is a full example of the allowed configuration options on Linux:
 	"ip-forward": false,
 	"ip-masq": false,
 	"userland-proxy": false,
+	"userland-proxy-path": "/usr/libexec/my-docker-proxy",
```

Comment:
ok, in which case I think `/usr/libexec/docker-proxy` is best as this would be the most common use case - not many people are going to override the proxy.